tests/lib/prepare.sh

```diff
@@ -3,7 +3,7 @@
 set -eux
 
 prepare_classic() {
-    apt install -y ${SPREAD_PATH}/../snapd_*.deb
+    dpkg -i ${SPREAD_PATH}/../snapd_*.deb || apt-get -f install -y
```

Comment:
I won't insist, it's a bit borderline, thanks for doing the functions for systemd-run though